tests/blueprints/tasks/github_summary_test.py

```diff
@@ -0,0 +1,76 @@
+import pytest
+from busy_beaver.blueprints.tasks.github_summary import start_post_github_summary_task
+
+MODULE_TO_TEST = "busy_beaver.blueprints.tasks.github_summary"
+
+
+@pytest.fixture
+def patched_post_github_summary_trigger(mocker, patcher):
+    return patcher(
+        MODULE_TO_TEST,
+        namespace=start_post_github_summary_task.__name__,
+        replacement=mocker.Mock(),
+    )
```

Comment:
Might make sense to use `functools.partial`, still figuring out best practices for this fixture. All I know right now is that it makes patching super simple.